client/profile-wizard/steps/business-details/flows/selective-bundle/selective-extensions-bundle/index.js

```diff
@@ -239,12 +235,52 @@ const BundleExtensionCheckbox = ( { onChange, description, isChecked } ) => {
 	);
 };
 
+/**
+ * Returns plugins that either don't have the acceptedCountryCodes param or one defined
+ * that includes the passed in country.
+ *
+ * @param {Array}  plugins  list of plugins
+ * @param {string} country  Woo store country
+ * @param {Array}  industry List of selected industries
+ */
+const getVisiblePlugins = ( plugins, country, industry ) => {
+	const countryCode = getCountryCode( country );
+
+	return plugins.filter(
+		( plugin ) =>
+			! plugin.isVisible || plugin.isVisible( countryCode, industry )
+	);
+};
+
 export const SelectiveExtensionsBundle = ( {
 	isInstallingActivating,
 	onSubmit,
+	country,
+	industry,
 } ) => {
 	const [ showExtensions, setShowExtensions ] = useState( false );
-	const [ values, setValues ] = useState( initialValues );
+	const [ values, setValues ] = useState( {} );
+
+	useEffect( () => {
+		const initialValues = installableExtensions.reduce(
+			( acc, curr ) => {
+				const plugins = getVisiblePlugins(
+					curr.plugins,
+					country,
+					industry
+				).reduce( ( pluginAcc, { slug } ) => {
+					return { ...pluginAcc, [ slug ]: true };
+				}, {} );
+
+				return {
+					...acc,
+					...plugins,
+				};
+			},
+			{ install_extensions: true }
+		);
```

Comment:
With the visibility parsing, it made more sense to write it in this format with a useEffect that depends on the `country`.